blockchain/genesis/genesis.go

```diff
@@ -112,9 +112,10 @@ func defaultConfig() Genesis {
 				Fee:          unit.ConvertIotxToRau(100).String(),
 				MinSelfStake: unit.ConvertIotxToRau(1200000).String(),
 			},
-			WithdrawWaitingPeriod: 3 * 24 * time.Hour,
-			MinStakeAmount:        unit.ConvertIotxToRau(100).String(),
-			BootstrapCandidates:   []BootstrapCandidate{},
+			WithdrawWaitingPeriod:            3 * 24 * time.Hour,
+			MinStakeAmount:                   unit.ConvertIotxToRau(100).String(),
+			BootstrapCandidates:              []BootstrapCandidate{},
+			EndorsementWithdrawWaitingBlocks: 24 * 60 * 60 / 5,
```

Comment:
what's the meaning of the value

Reply:
After submitting the endorsement withdraw action, it takes the waiting blocks for it to take effect. During this period, the delegate can continue to participate in consensus, and the bucket is locked, meaning it cannot be unstaked or voted to others.